lib/coffeescript/kits/paged-publication/core.coffee

```diff
@@ -172,10 +172,12 @@ class PagedPublicationCore
 
     beforeNavigation: (e) ->
         position = e.newPosition
-        versoPageSpread = @getVerso().getPageSpreadFromPosition position
+        theVerso = @getVerso()
```

Comment:
Odd naming IMO.

Reply:
`verso` key was already taken up by the event info, what's another alternative? `versoInstance`?

Reply:
The variable 'verso' isn't used in this function as far as I can see?

Reply:
No but that name is used for a key on the `beforeNavigation` event object and I wanted to avoid any confusion

Reply:
Can't you use e.verso instead of @getVerso() then?

Reply:
Are you suggesting we include the verso instance in the events from `verso-browser`? It's not right now in any case, might be widely useful? https://github.com/shopgun/verso-browser/blob/develop/lib/coffeescript/verso.coffee#L119

Reply:
Ah, right. What do you think? Might be a tad nasty with instances in event data but we already do it with other instances so might not be an issue. Otherwise, `theVerso` is fine for now.

Reply:
Can we just uniformly have the verso instance be the second argument to all events from verso? Does microevent let us do that?

Feel free to merge and maybe create an issue over on shopgun/verso-browser